### PR TITLE
auto case OCP-46637

### PR DESCRIPTION
**Pull request by huiran0826**
auto case OCP-46637 which is from customer bug: https://bugzilla.redhat.com/show_bug.cgi?id=2024880

Test log: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/3815/console

/cc @openshift/team-sdn-qe

### CODE REVIEW

File: features/networking/egress-ip.feature [hunk 1]
```diff
@@ -869,3 +869,47 @@ Feature: Egress IP related features
...
+  # @case_id OCP-46637 
+  @admin
+  @destructive
+  @4.10 @4.9
+  @vsphere-ipi
+  @vsphere-upi
+  Scenario: [Bug 2024880] EgressIP should work when configuring networkpolicy
+    Given I save ipecho url to the clipboard
+    Given I store the schedulable workers in the :workers clipboard
+    And the valid egress IP is added to the "<%= cb.workers[0].name %>" node
```
Comment (weliang1) [inline features/networking/egress-ip.feature:883]: This test egressIP for SDN cluster, not sure if we need cover same egressIP + networkpolicy testing for OVN cluster.

Comment (huiran0826) [reply features/networking/egress-ip.feature:883]: yeah, that would be seperate case for OVN (not in this PR) as we know the different implementation between sdn and ovn.

Comment (weliang1) [reply features/networking/egress-ip.feature:883]: /lgtm

**Pull request merged**